.travis.yml

```diff
@@ -71,6 +71,10 @@ jobs:
     bundler_args: --without development release
     env: BEAKER_PUPPET_COLLECTION=puppet6 BEAKER_setfile=ubuntu1804-64 CHECK=beaker
     services: docker
+  - rvm: 2.5.3
```

Comment:
The task for Puppet 5 is missing.

Reply:
Because there are no puppet 5 packages for ubuntu 20